### PR TITLE
Add tips buttons. Link to docs.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
screen shot 2019-02-22 at 12 21 09 pm
screen shot 2019-02-22 at 12 21 26 pm
screen shot 2019-02-22 at 12 20 52 pm
screen shot 2019-02-22 at 12 21 45 pm

### CODE REVIEW

File: packages/devtools/lib/src/debugger/debugger.dart [hunk 1]
```diff
@@ -163,8 +163,12 @@ class DebuggerScreen extends Screen {
                         ])
                         ..small(),
                     ]),
-                  div()..flex(),
                   breakOnExceptionControl,
+                  div()..flex(),
+                  div(c: 'btn-group margin-left')
+                    ..add([
+                      createTipsButton('Debugger', 'debugger'),
```
Comment (jacob314) [inline packages/devtools/lib/src/debugger/debugger.dart]: all these cases the two fields are the same modulo camel case. 
Go ahead and just pass the one name and lower case to generate the url.

File: packages/devtools/lib/src/ui/ui_utils.dart [hunk 2]
```diff
@@ -73,6 +74,17 @@ List<CoreElement> getServiceExtensionButtons() {
   ];
 }
 
+CoreElement createTipsButton(String pageName, String endPath) {
```
Comment (jacob314) [inline packages/devtools/lib/src/ui/ui_utils.dart]: nit change to just have a single
pageName
and remove endPath
make endPath just be pageName.toLowerCase() or similar.

**Comment by devoncarew** [comment]
drive by comment: I wonder if we should make them slightly less button like? They'll act more like links than buttons.

File: packages/devtools/lib/src/debugger/debugger.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools/lib/src/debugger/debugger.dart]: done.

File: packages/devtools/lib/src/ui/ui_utils.dart (hunk 2 above)
Comment (kenzieschmoll) [reply packages/devtools/lib/src/ui/ui_utils.dart]: done.

**Comment by kenzieschmoll** [comment]
Good point, Devon. I'll land this and we can revisit the design.

**Pull request merged**